### PR TITLE
Replace udunits2::ud.is.parseable with PEcAn.utils::unit_is_parseable

### PR DESCRIPTION
**Pull request by nanu1605**

### CODE REVIEW

File: base/utils/R/unit_is_parseable.R [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+##' @return TRUE if the units is parseable, FALSE otherwise.
+##' @export
+unit_is_parseable <- function(u1){
+  tryCatch({
+    if(units::as_units(u1))
+      return(TRUE)
+  },
+  error = function(e) FALSE
+  )
+} # unit_is_parseable
```
Comment (moki1202) [inline base/utils/R/unit_is_parseable.R:17]: @nanu1605 go ahead and add a newline here. Rest everything looks good and clean to me. Let's wait for other reviewers and once we're good, I think you can go ahead and replace the `udunits2::ud.is.parseable()` function in your previous PRs.

Comment (moki1202) [inline base/utils/R/unit_is_parseable.R:17]: @nanu1605 go ahead and add a newline here. Rest everything looks good and clean to me. Let's wait for other reviewers and once we're good, I think you can go ahead and replace the `udunits2::ud.is.parseable()` function in your previous PRs.

**Review by Aariq — APPROVED** [review]
Looks good to me.  Is the plan to swap out `ud.is.parseable` in other PEcAn functions later/as needed?

**Review by infotroph — APPROVED** [review]

**Comment by nanu1605** [comment]
As soon as this PR is merged i will start working on replacing `ud.is.parseable`

**Pull request merged**